tests/runtime/TestNegativeIGBADetection.hip

```diff
@@ -0,0 +1,14 @@
+// Check we detect that the device code does not have indirect global
```

Comment:
Pls add the copyright/license texts to new files.

Reply:
Done. Is it fine to use "short-form" copyright text? And was it the MIT license the chipStar is using?

Reply:
The existing LICENSE file in project root seems to match to the MIT license.